README.md

```diff
@@ -0,0 +1,73 @@
+My Blackjack game uses 6 decks for a total of 312 cards, once below 50% the deck will be reshuffled. 
+
+
+# Technologies Used 
+<img src="https://img.shields.io/badge/JavaScript-323330?style=for-the-badge&logo=javascript&logoColor=F7DF1E"/>
+<img src="https://img.shields.io/badge/CSS3-1572B6?style=for-the-badge&logo=css3&logoColor=white"/>
+<img src="https://img.shields.io/badge/HTML5-E34F26?style=for-the-badge&logo=html5&logoColor=white"/>
+<img src="https://img.shields.io/badge/GIT-E44C30?style=for-the-badge&logo=git&logoColor=white"/>
+
+# Credits 
+SVG generator
+[https://sean.brunnock.com/SVG/SVGPathGenerator/]<br>
+Curved text guide
+[https://css-tricks.com/snippets/svg/curved-text-along-path/]<br>
+Gradient text guide
+[https://fossheim.io/writing/posts/css-text-gradient/]<br>
+Gradient maker
+[https://cssgradient.io/]<br>
+Deal card audio
+[https://freesound.org/people/egomassive/sounds/536784/]<br>
+Slide card audio
+[https://freesound.org/people/el_boss/sounds/571574/]<br>
+Clapping audio
+[https://freesound.org/people/timtube/sounds/57733/]<br>
+Tada fanfare G
+[https://freesound.org/people/plasterbrain/sounds/397353/]<br>
+Casino chip audio
+[https://freesound.org/people/strangehorizon/sounds/618760/]<br>
+MichaelNo audio from The Office 
+[https://www.voicy.network/clips/NVurDahoEeihTtSuUqiQGg]<br>
+Confentti courtesy of https://github.com/mathusummut/
+[https://github.com/SEI-Remote/confetti]<br>
+Animate
+[https://animate.style/]
+CSS cardstarter from GA
+[https://github.com/SEI-Remote/flippin-awesome-improved]
+
+# Ice Box / Stretch Goals
+- [x] Sound effects
+- [ ] Card Animation
+- [ ] Dealer portrait that updates depending on what is going on
+- [ ] Show the deck and discard deck for continuity sake
+- [ ] Mobile accessibility
+- [ ] More players or basic AI players to add more authenticity
+- [x] Betting
+- [X] Chip Image for bets
+- [x] Double
+- [ ] Split / Surrender / and even maybe Insurance
+- [ ] Add Fisher-Yates Shuffle or other shuffle if better
+
+# Wireframe and Design
+![picture of my wireframe](./images/wireframe.png)
+Original wireframe, definetely thought simple.
+[https://trello.com/b/L9ryi2dz/euril-blackjack]
+![Ver 1.0 of game](./images/blackjackRM.png)
+![Ver 1.0 of game in play](./images/blackjackRM2.png)
+Hope you liked my version of blackjack, I enjoyed thinking out how to translate playing the game into javascript code. I went out of my way to mimic a blackjack table, I made the background using the gradient tool then bringing that into a clone of my code to make things show up where I wanted them to. I really like how the curved text/lines turned out, they ended up being in the right location for the player hand cards to spawn. I made the casino chips in gimp.
```

Comment:
Suggested edits to this section:
I hope you liked my version of Blackjack, I enjoyed thinking out how to translate playing the game into JavaScript. I went out of my way to mimic a blackjack table. I made the background using the gradient tool and then brought it into a clone of my code to make things show up where I wanted them to. I really like how the curved text/lines turned out; they ended up being in the right location for the player hand cards to spawn. I made the casino chips in gimp. Overall my inspiration for the design came from games I've played Blackjack in as I've never actually played Blackjack in a casino before. Shoutout to the Yakuza series and Fallout: New Vegas.